packages/synapse-sdk/README.md

```diff
@@ -42,6 +42,24 @@ Note: `ethers` v6 is a peer dependency and must be installed separately.
 
 Check the documentation [website](https://synapse.filecoin.services/)
 
+## Telemetry
+
+Telemetry is enabled by default to help us validate functionality while moving towards GA launch. Disable it at runtime if you prefer:
```

Comment:
We need to be super up-front and clear about this, it's going to be a very sensitive topic for much of our user-base.

```suggestion
To help us validate functionality and iron out problems throughout the whole stack, starting from the SDK, telemetry is **enabled by default** in Synapse. This will be the case during the current development period prior to our upcoming General Availability launch.

You are welcome to disable it in the runtime when you use Synapse if you would prefer to opt out:
```

Can we say anything more about what's collected, that it's "not personally identifying information" or anything like that?